src/tree/explorerTree.ts

```diff
@@ -29,6 +37,17 @@ export class ExplorerTree implements TreeDataProvider<BasicTreeItem> {
     this.refresh()
   }
 
+  private readonly activeProjectListener = async () => {
+    const currentUserId = configVaultService.getCurrentUserID()
+    const activeProjectId = configVaultService.getActiveProjectId(currentUserId)
+    const projects = await iamState.listProjects()
+
+    if (projects) {
```

Comment:
`projects` always return an array and even if it is empty it will return true
change to 
```ts
if (projects.length > 0) { ...
```

Reply:
fixed